mavros_extras/msg/DistanceSensor.msg

```diff
@@ -0,0 +1,14 @@
+# DISTANCE_SENSOR message data
+
+Header header
+
+sensor_msgs/Range range
+
+uint8 type			# see MAV_DISTANCE_SENSOR enum
+uint8 MAV_DISTANCE_SENSOR_LASER = 0
+uint8 MAV_DISTANCE_SENSOR_ULTRASOUND = 1
+
+uint8 id 			# XXX FIXME enum ??
```

Comment:
In this case, we are translating the message from the MAVLink frame to the ROS msg, so it should be the other way (i think): knowing the type of data received in the MAVLink frame, it should define what's the sensor frame, and as a consequence, the `radiation_type`. Correct me if I'm wrong.